src/node/scripts/fetch-prebuild.js

```diff
@@ -51,7 +52,15 @@ async function downloadIfNeeded() {
 function download() {
   console.log(`downloading ${URL}`);
   return new Promise((resolve, reject) => {
-    https.get(URL, async res => {
+    let options = {};
+    if (process.env.HTTPS_PROXY != undefined) {
+      options.agent = new HttpsProxyAgent(process.env.HTTPS_PROXY);
+    }
+    if (process.env.https_proxy != undefined) {
+      options.agent = new HttpsProxyAgent(process.env.https_proxy);
+    }
```

Comment:
Can we keep only `HTTPS_PROXY` and remove `https_proxy`?

Reply:
Fixed! Thanks!